packages/transit-vehicle-overlay/src/index.story.js

```diff
@@ -133,7 +135,7 @@ function rectangles(popup = true) {
   // give action to the popup vehicle tracking button
   VehiclePopup.defaultProps.setTracked = (vehicle, isTracked) => {
     updateTrackedVehicle(vehicle, isTracked);
-    if (!isTracked) recenter(null, 0, 0); // no longer tracking so clear out cached coords
+    if (!isTracked) recenter(null, 0, doTracking, 0); // no longer tracking so clear out cached coords
```

Comment:
Is the `doTracking` argument be just `false` for vehicles that are no longer tracked?